test/test_engine_ngspice.c

```diff
@@ -195,29 +351,28 @@ static void test_engine_ngspice_error_step_zero() {
 	gsize expected_size;
 	g_file_get_contents(expected_file, &expected_content, &expected_size, NULL);
 
-	//
-	// g_assert_cmpstr(actual_content, ==, expected_content);
-  //
-  // const gchar *array[] = {
-  // 			"\n",
-  // 			"ngspice stopped due to error, no simulation run!\n",
-  // 			"\n",
-  // 			"ERROR: fatal error in ngspice, exit(1)\n",
-  // 			"### ngspice exited abnormally ###\n",
-  // 			"### netlist error detected ###\n",
-  // 			"You made a mistake in the simulation settings or part properties.\n",
-  // 			"The following information will help you to analyze the error.\n",
-  // 			NULL
-  // 	};
-  //
-  // 	GList *walker = test_resources->log_list;
-  //
-  // 	for (int i = 0; array[i] != NULL; i++) {
-  // 		g_assert_nonnull(walker);
-  // 		g_assert_nonnull(walker->data);
-  // 		g_assert_cmpstr(walker->data, ==, array[i]);
-  // 		walker = walker->next;
-  // 	}
+	g_assert_cmpstr(actual_content, ==, expected_content);
+
```

Comment:
comparing for equality is no solution, I can imaginge that even arch specific rounding might cause this to fail

Reply:
`_cmpstr`, so it's fine.